### PR TITLE
Completed Activity Feed and Manage Orgs

### PR DESCRIPTION
**Pull request by RiddhiGupta5**

### CODE REVIEW

File: org-monster-extension/manage-orgs.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  chrome.storage.sync.set({ org: orgs }, function () {
+    addTag(orgName);
+    addcloseEventListener();
+  });
+}
+
+// Adding Tags
+function addTag(orgName) {
+  var orgsList = document.getElementById("orgsList");
+  orgsList.innerHTML += `<li class="org-tag"><span class="tag-text">${orgName}</span><span class="close">&times;</span></li>`;
```
Comment (L04DB4L4NC3R) [inline org-monster-extension/manage-orgs.js:99]: XSS vulnerability alert. You are directly using user input on the same HTML without sanitizing it.

**Review by L04DB4L4NC3R — CHANGES_REQUESTED** [review]

**Comment by Geek-ubaid** [comment]
@RiddhiGupta5 some changes still left to be done, PR getting stale

**Comment by RiddhiGupta5** [comment]
> @RiddhiGupta5 some changes still left to be done, PR getting stale

Yeah will do the changes by tonight

**Review by Geek-ubaid — APPROVED** [review]

**Comment by Geek-ubaid** [comment]
@L04DB4L4NC3R review the changes and merge the PR

**Review by L04DB4L4NC3R — APPROVED** [review]

**Pull request merged**